platform/security/servlet/security-servlet-logout/src/main/java/org/codice/ddf/security/servlet/logout/LocalLogoutServlet.java

```diff
@@ -50,14 +45,11 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
 
     boolean mustCloseBrowser = (checkForBasic(request) || checkForPki(request));
 
-    String message =
-        String.format(
-            "{ \"mustCloseBrowser\": %b, \"redirectUrl\": \"%s\" }",
-            mustCloseBrowser, getRedirectUrlMessage());
+    String message = String.format("{ \"mustCloseBrowser\": %b }", mustCloseBrowser);
 
-    response.setStatus(HttpServletResponse.SC_OK);
-    response.setContentType("application/json");
     try {
+      response.setStatus(HttpServletResponse.SC_OK);
+      response.setContentType("application/json");
```

Comment:
❓ These setters can't throw `IOException`s, correct?

Reply:
They cannot. The modifications to LocalLogoutServlet in this PR are reverting the module back to how it was before any work was done for the original logout redirect PR. I can keep the change fromt the old PR that takes these lines out of the try/catch